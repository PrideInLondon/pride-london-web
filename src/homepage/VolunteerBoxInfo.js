import React, { Fragment } from 'react'
import { Container, Row, Column } from '../components/grid'
import { Button } from '../components/button'
import {
  ContentVolunteer,
  BackgroundVolunteer,
} from './VolunteerBoxInfo.styles'

const VolunteerBoxInfo = () => (
  <Fragment>
    <BackgroundVolunteer />
    <Container>
      <Row>
        <Column>
          <ContentVolunteer>
            <h2>Volunteer</h2>
            <p>
              Pride in London is run almost entirely by volunteers. We could use
              people like you to help us organise the best event that we can.
            </p>
            <Button to="https://volunteer.prideinlondon.org">
              Help keep Pride going
            </Button>
          </ContentVolunteer>
        </Column>
      </Row>
    </Container>
  </Fragment>
)

export default VolunteerBoxInfo
