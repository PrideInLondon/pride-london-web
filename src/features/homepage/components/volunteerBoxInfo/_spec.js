import React from 'react'
import { shallow } from 'enzyme'
import VolunteerBoxInfo from '.'

describe(VolunteerBoxInfo.name, () => {
  it('volunteer snapshot', () => {
    const wrapper = shallow(<VolunteerBoxInfo />)
    expect(wrapper).toMatchSnapshot()
  })
})
