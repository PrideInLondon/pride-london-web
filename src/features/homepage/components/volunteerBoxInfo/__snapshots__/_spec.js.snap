// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VolunteerBoxInfo volunteer snapshot 1`] = `
<Fragment>
  <styles__BackgroundVolunteer />
  <grid__Container
    mx="auto"
  >
    <Row>
      <Column>
        <styles__ContentVolunteer>
          <h2>
            Volunteer
          </h2>
          <p>
            Pride in London is run almost entirely by volunteers. We could use people like you to help us organise the best event that we can.
          </p>
          <Button
            className=""
            disabled={false}
            flexwidth={false}
            fullmobile={false}
            link={false}
            onClick={null}
            primary={true}
            small={false}
            to="#"
            type="button"
            white={false}
            wide={false}
          >
            Help keep Pride going
          </Button>
        </styles__ContentVolunteer>
      </Column>
    </Row>
  </grid__Container>
</Fragment>
`;
